pkg/runner/cluster_k8s.go

```diff
@@ -110,9 +104,13 @@ type ClusterK8sRunnerConfig struct {
 	// Provider is the infrastructure provider to use
 	Provider string `toml:"provider"`
 
-	// Resources requested for each pod from the Kubernetes cluster
-	PodResourceMemory string `toml:"pod_resource_memory"`
-	PodResourceCPU    string `toml:"pod_resource_cpu"`
+	// Resources requested for each testplan pod from the Kubernetes cluster
```

Comment:
This was supposed to be addressed in https://github.com/testground/testground/pull/1008